christfellowshipchurch/web-app-v2
# Career pages

### About
This is the Careers Page PR for both the Job Categories Page and Job Details Page 

### Test Instructions
1. Open /careers to see the Job Categories Page, you can test the UI against the[ Figma design ](https://www.figma.com/file/D9VPx0A14ayYjRIDbR5oLu/Careers-Page?node-id=0%3A1&t=PBOv89xt9UjIXNS8-1)
2. Click on any of the jobs from there and you'll see the Job Details Page. 
3. Make sure all spacing looks right in all devices and that all links/buttons etc work. 

Notes: 
We still don't have the actual images so these are dummy ones. 
We ran into an issue with the Application page and will talk to Brandi about it but that's probably going to be another ticket.

### Screenshots
<img width="1440" alt="Screen Shot 2022-12-09 at 9 32 36 AM" src="https://user-images.githubusercontent.com/46769629/206725149-ccb2b742-3ffa-4d07-81e6-7fa0f2271b65.png">

<img width="1436" alt="Screen Shot 2022-12-09 at 9 31 59 AM" src="https://user-images.githubusercontent.com/46769629/206725025-570dcaed-a04d-467e-8caf-955a6cf3a69f.png">
content.com/46769629/206722478-a6d7ee44-f924-4d46-87a3-0e8c64fec444.png">


### Closes Tickets
[CFDP-2314](https://christfellowshipchurch.atlassian.net/browse/CFDP-2314)
[CFDP-2315](https://christfellowshipchurch.atlassian.net/browse/CFDP-2315)



[CFDP-2314]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2314?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
